PYME/Acquire/Hardware/Coherent/OBIS.py

```diff
@@ -138,6 +141,35 @@ def SetPower(self, power):
     def GetPower(self):
         return self.power
 
+    def SetOpMode(self,mode):
+        """
+        mode = 0: CWP, CW with constant power
```

Comment:
Arbitrary integer constants doesn't make for very readable code (where this is being called). Having class level constants, e.g. `CoherentOBISLaser.OBIS_MODE_CW` is better style / more readable in calling code ...

compare:

```python
my_laser.SetOpMode(2) # Doesn't convey what this is doing without looking at the laser code
```
with
```python
m_laser.SetOpMode(my_laser.OBIS_MODE_ANALOG_MOD)
```
The other pythonic way of doing it would be to take a string, e.g. 'CW', 'ANALOG_MOD', etc ... and do the translation into mode numbers in the method.